### ISSUE TITLE
Error in standalone tests with python 2.7

### ISSUE THREAD

**Issue by ddatsko** [Nuitka/Nuitka#352]
*  Nuitka version (develop version of Nuitka installed from git)
```
0.6.4rc2
Python: 3.7.3 (default, Mar 26 2019, 21:43:19) 
Executable: /usr/bin/python
OS: Linux
Arch: x86_64
```
*  When I run standalone tests (Nuitka/tests/standalone/run_all.py) with Python 2.7 I get an error:
```
Using concrete python 2.7.16 on x86_64
Consider output of recursively compiled program: CtypesUsing.py
Comparing output of 'CtypesUsing.py' using '/usr/bin/python2' with flags silent, expect_success, standalone, remove_output, plugin_disable:pylint-warnings ...
--- /usr/bin/python2 (stdout)
+++ nuitka (stdout)

@@ -1,2 +1 @@

-OK.
 
--- /usr/bin/python2 (stderr)
+++ nuitka (stderr)

@@ -1 +1,4 @@

+Traceback (most recent call last):
+  File "/home/denys/Nuitka/tests/standalone/CtypesUsing.dist/CtypesUsing.py", line 20, in <module>
+ImportError: No module named ctypes
 
Exit codes 0 (CPython) != 1 (Nuitka)
Error, outputs differed.
```

**Comment by kayhayen** [comment]
What OS is this?

**Comment by ddatsko** [comment]
Arch Linux

**Comment by kayhayen** [comment]
@ddatsko Ok, I will have to update my Arch to latest, lets see how that goes. :)

**Comment by kayhayen** [comment]
The last release contains a fix, where on macOS it was discovered that the module filename string was not properly terminated. It was never observed on Linux, but I think this issue potentially did. Sorry for not getting back to it at the time. But I think this is fixed with 0.6.5 now.
